tests/odov3/go.mod

```diff
@@ -1,6 +1,6 @@
 module github.com/devfile/registry/tests/odov3
 
-go 1.19
+go 1.21
 
 require (
 	github.com/devfile/api/v2 v2.2.2
```

Comment:
This is why I left this PR in WIP / blocked, need https://github.com/devfile/library/pull/212 to merge before I can update devfile/api since devfile/library is a direct dependency as well.

Reply:
> This is why I left this PR in WIP / blocked, need [devfile/library#212](https://github.com/devfile/library/pull/212) to merge before I can update devfile/api since devfile/library is a direct dependency as well.

Oh I see its merged now, I'll update the dependencies today.

Reply:
Also I didn't notice while reviewing that the PR was actually draft :D